src/League/ColorExtractor/Color.php

```diff
@@ -24,4 +24,28 @@ public static function fromHexToInt($color)
     {
         return hexdec(ltrim($color, '#'));
     }
+
+    /**
+     * @param string $color
+     *
+     * @return array
+     */
+    public static function fromHexToRGB($color)
+    {
+        list($r, $g, $b) = sscanf($color, "#%02x%02x%02x");
+
+        return ['r' => $r, 'g' => $g, 'b' => $b];
+    }
+
+    /**
+     * @param int $color
+     *
+     * @return array
+     */
+    public static function fromIntToRGB($color)
```

Comment:
You could use bit shifting instead of intermediate conversion. Like red component is `$color >> 16 & 0xFF`